### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by timrowledge**
some typos.
The PiFace flickering bug has been fixed.
Add CamJam motor board info
Re-add PiGLow section
Extend examples of commands

### CODE REVIEW

**Comment by bennuttall** [comment]
Thanks @timrowledge.

There's a command you've given to control the Ryanteck MCB, but you've spelled it differently. The header says "Ryanteck" (correct) but the command says `RyanTek001` (incorrect). Is this just a typo in the docs, or is it misspelled in the implementation?

**Comment by bennuttall** [comment]
@lauraclay This page went in a few months ago and could do with a once-over when you get chance. Thanks

File: usage/scratch/gpio/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The basic gpio layer command list is
     + outputpwm to set as a pwm pin
 example `config12in`  
 - gpio + pin number + [ on | high | off | low ]
-example `gpio17on`
+example `gpio17on`p
```
Comment (lurch) [inline usage/scratch/gpio/README.md]: Was the 'p' added here a typo?

**Comment by timrowledge** [comment]
Eek, yes the 'ryantek' is a typo in the software as well as the doc. Damn. Still, it's easy to fix so that either 'RyanTek001' or 'RyanTeck001' will work.

And there extraneous 'p' is indeed a typo, too. All Hail Tpyos, Goddess of spilling mistaichs!

**Pull request merged**